libvuln/jsonblob/jsonblob.go

```diff
@@ -103,6 +103,9 @@ func (l *Loader) Next() bool {
 			}
 		}
 	}
+	if l.Err() != nil {
```

Comment:
this may not be completely correct. This allows EOF to still return `true` which is probably ok, but only if `l.next != nil`, I think

Reply:
By my thinking, the error check on the first entry should catch this?